doc/man1/s_server.pod

```diff
@@ -709,6 +710,13 @@ greater than or equal to 0.
 
 Accept early data where possible.
 
+=item B<-no_anti_replay>
+
+Switches off replay protection. With replay protection on OpenSSL will
+automatically detect if a session ticket has been used more than once, TLSv1.3
+has been neotiated and early data is enabled on the server. A full handshake is
+forced if a session ticket is used a second or subsequent time.
```

Comment:
Perhaps the text should explain what happens to the early data sent along with such a replay... Is it accepted? Ignored? Does the client have to re-sendt it after the full handshake?

Reply:
I added a note about early data here.